pkg/policy/policiesSchema.json

```diff
@@ -0,0 +1,78 @@
+{
+  "type": "object",
+  "properties": {
+    "apiVersion": {
+      "type": "string",
+      "enum": [
+        "v1"
+      ]
+    },
+    "message": {
+      "type": "string",
+      "enum": [
+        "Should have exact one policy set as default",
+        "All policy names should be unique",
+        "All rule identifiers should be unique"
+      ]
+    },
+    "policies": {
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "type": "object",
+        "required": [
+          "name"
+        ],
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "isDefault": {
+            "type": "boolean"
+          },
+          "message": {
+            "type": "string"
+          },
+          "rules": {
+            "items": {
+              "type": "object",
+              "properties": {
+                "identifier": {
+                  "type": "string",
+                  "pattern": "^([A-Z0-9])+(_?[A-Z0-9])*$"
```

Comment:
to the best of my knowledge, we don't have this regex limitation in our backend validator